lib/checks/isQuadratic.js

```diff
@@ -12,13 +12,14 @@ function isQuadratic(node) {
     return false;
   }
 
+  // get the number of symbols in the expression and ensure there's only one
```

Comment:
sorry I'm being really picky now so you don't have to change this but

I think this comment is pretty close to what the code is doing, I meant more like 
"make sure only one symbol appears in the expression, i.e. just x and not both x and y"

the point of confusion for people could be that it's a _set_ (which you show in the var name which is helpful, so it's probably fine) and there can be multiple symbol terms (multiple symbols) as long as they're of the same symbol name